services-js/registry-certs/client/birth/questions/BornInBoston.tsx

```diff
@@ -1,34 +1,107 @@
 import React from 'react';
 
+import QuestionComponent from './QuestionComponent';
 import FieldsetComponent from './FieldsetComponent';
 import YesNoUnsureComponent from './YesNoUnsureComponent';
 
 import { YesNoUnknownAnswer } from '../QuestionsFlow';
 
 interface Props {
   forSelf: boolean | null;
-  currentValue: YesNoUnknownAnswer;
+  bornInBoston?: YesNoUnknownAnswer;
+  parentsLivedInBoston?: YesNoUnknownAnswer;
 
-  handleChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  handleProceed: (answers: Partial<State>) => void;
```

Comment:
You can take off the `Partial` here because you will be calling this function with all of `State`.